### DISCUSSION TITLE
Hover Styles Not Being Applied to Button

### DISCUSSION THREAD

**Discussion by favorodera** [primefaces/community#2430] [PrimeVue] [answered]
The hover styles is not being applied instead shows the default hover styles. (well unless i apply the unstyled prop)
Am i using the pass through correctly 

`   <PrimeButton as="router-link" label="Home" to="/" icon="i-fa6-solid-house" text 
      :pt="{ root: { class: 'group justify-self-start bg-transparent border-0 flex items-center gap-1 p-2 ' }, label:{class:'text-#4859f3 font-500 text-4 opacity-0 group-hover:opacity-100 transition-opacity transition-delay-50 transition-ease transition-duration-1000 '}, icon:{class:' group-hover:text-#4859f3 w-5 h-5 transition-color transition-delay-50 transition-ease transition-duration-1000'} }" />`

**Comment by j0rgedev** [comment, marked as answer]
Hello @favorodera. I've just tested the code you shared. Since you want to set your own styles using the pt attribute, you must set the unstyled prop to true. This will tell PrimeVue to ignore the built-in styles and apply yours. Read further in [PrimeVue Unstyled Mode Example](https://primevue.org/theming/unstyled/#example)

**Reply by favorodera** [reply]
Ok, Thanks
